### PR TITLE
app-admin/mongosh-bin: bump to 1.0.7

### PR DESCRIPTION
**Pull request by hydrapolic**
Signed-off-by: Tomáš Mózes <hydrapolic@gmail.com>

### CODE REVIEW

File: app-admin/mongosh-bin/mongosh-bin-1.0.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline app-admin/mongosh-bin/mongosh-bin-1.0.7.ebuild:4]: 8 can be used already ;)

File: app-admin/mongosh-bin/mongosh-bin-1.0.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		arm64? ( ${SRC_URI_BASE}-arm64.tgz -> ${MY_P}_arm64.tgz )"
+
+LICENSE="Apache-2.0 BSD BSD-2 CC-BY-4.0 ISC MIT WTFPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+RDEPEND="dev-libs/cyrus-sasl[kerberos]"
+
+# Do not complain about CFLAGS etc since we don't use them
+QA_FLAGS_IGNORED='.*'
```
Comment (juippis) [inline app-admin/mongosh-bin/mongosh-bin-1.0.7.ebuild:23]: I'm somewhat surprised this works.

**Review by juippis — APPROVED** [review]

File: app-admin/mongosh-bin/mongosh-bin-1.0.7.ebuild (hunk 2 above)
Comment (hydrapolic) [reply app-admin/mongosh-bin/mongosh-bin-1.0.7.ebuild:23]: Taken from go-module.eclass.

**Comment by hydrapolic** [comment]
Thanks @juippis

**Pull request closed without merging**